api/src/main/java/org/apache/iceberg/types/Types.java

```diff
@@ -492,6 +517,14 @@ public String doc() {
       return doc;
     }
 
+    public Object initialDefaultValue() {
```

Comment:
Do we need `Value` here? It seems redundant. Maybe just `initialDefault` and `writeDefault`.